video/out/vo_gpu_next.c

```diff
@@ -1396,27 +1386,36 @@ static void update_render_options(struct priv *p)
     p->peak_detect.scene_threshold_low = opts->tone_map.scene_threshold_low;
     p->peak_detect.scene_threshold_high = opts->tone_map.scene_threshold_high;
 
-    static const enum pl_tone_mapping_algorithm tone_map_algos[] = {
-        [TONE_MAPPING_CLIP]     = PL_TONE_MAPPING_CLIP,
-        [TONE_MAPPING_MOBIUS]   = PL_TONE_MAPPING_MOBIUS,
-        [TONE_MAPPING_REINHARD] = PL_TONE_MAPPING_REINHARD,
-        [TONE_MAPPING_HABLE]    = PL_TONE_MAPPING_HABLE,
-        [TONE_MAPPING_GAMMA]    = PL_TONE_MAPPING_GAMMA,
-        [TONE_MAPPING_LINEAR]   = PL_TONE_MAPPING_LINEAR,
-        [TONE_MAPPING_BT_2390]  = PL_TONE_MAPPING_BT_2390,
+    static const struct pl_tone_map_function * const tone_map_funs[] = {
+        [TONE_MAPPING_AUTO]     = &pl_tone_map_auto,
+        [TONE_MAPPING_CLIP]     = &pl_tone_map_clip,
+        [TONE_MAPPING_MOBIUS]   = &pl_tone_map_mobius,
+        [TONE_MAPPING_REINHARD] = &pl_tone_map_reinhard,
+        [TONE_MAPPING_HABLE]    = &pl_tone_map_hable,
+        [TONE_MAPPING_GAMMA]    = &pl_tone_map_gamma,
+        [TONE_MAPPING_LINEAR]   = &pl_tone_map_linear,
+        [TONE_MAPPING_SPLINE]   = &pl_tone_map_spline,
+        [TONE_MAPPING_BT_2390]  = &pl_tone_map_bt2390,
+        [TONE_MAPPING_BT_2246A] = &pl_tone_map_bt2446a,
```

Comment:
Pretty much all the references in mpv are typo'd as 2246 instead of 2446.

Reply:
Fixed, thanks for noticing